executionserver/image.yaml

```diff
@@ -25,6 +25,9 @@ envs:
       value: "true"
     - name: AB_JOLOKIA_HTTPS
       value: "true"
+    - name: "KIE_SERVER_ID"
+      example: "autogenerated-id"
```

Comment:
In BPMS 7.x binary there is default Kie server id set as "default-kieserver". I think it could be same here, to keep same user experience.